spring-security-cors/src/main/java/com/baeldung/springbootsecuritycors/controller/ResourceController.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.springbootsecuritycors.controller;
+
+import javax.servlet.http.HttpServletRequest;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@CrossOrigin
+public class ResourceController {
+
+    @RequestMapping("/user")

```

Comment:
It's more secure to declare which method explicitly, so let's use @GetMapping

Reply:
Changed to @GetMapping